switchapi/switch_nhop_int.h

```diff
@@ -67,36 +67,36 @@ extern "C" {
         _device, SWITCH_HANDLE_TYPE_NHOP, _handle, (void **)_info); \
   })
 
-#define switch_ecmp_member_handle_create(_device) \
+#define switch_nhop_member_handle_create(_device) \
   switch_handle_create(                           \
-      _device, SWITCH_HANDLE_TYPE_ECMP_MEMBER, sizeof(switch_ecmp_member_t))
+      _device, SWITCH_HANDLE_TYPE_NHOP_MEMBER, sizeof(switch_nhop_member_t))
 
-#define switch_ecmp_member_handle_delete(_device, _handle) \
-  switch_handle_delete(_device, SWITCH_HANDLE_TYPE_ECMP_MEMBER, _handle)
+#define switch_nhop_member_handle_delete(_device, _handle) \
+  switch_handle_delete(_device, SWITCH_HANDLE_TYPE_NHOP_MEMBER, _handle)
 
-#define switch_ecmp_member_get(_device, _handle, _info)                    \
+#define switch_nhop_member_get(_device, _handle, _info)                    \
   ({                                                                       \
-    switch_ecmp_member_t *_tmp_ecmp_member_info = NULL;                    \
-    (void)(_tmp_ecmp_member_info == *_info);                               \
+    switch_nhop_member_t *_tmp_nhop_member_info = NULL;                    \
+    (void)(_tmp_nhop_member_info == *_info);                               \
     switch_handle_get(                                                     \
-        _device, SWITCH_HANDLE_TYPE_ECMP_MEMBER, _handle, (void **)_info); \
+        _device, SWITCH_HANDLE_TYPE_NHOP_MEMBER, _handle, (void **)_info); \
   })
 
-#define switch_ecmp_group_get(_device, _handle, _info)                    \
+#define switch_nhop_group_get(_device, _handle, _info)                    \
```

Comment:
same as line 47 (verb first) so switch_get_nhop_group to be consistent overall in the code